### PR TITLE
Resumeable download feature has been enabled

### PR DESCRIPTION
**Pull request by praveen2gemini**

### CODE REVIEW

File: ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadRequest.java [hunk 1]
```diff
@@ -211,6 +221,14 @@ public boolean getDeleteDestinationFileOnFailure() {
         return mDeleteDestinationFileOnFailure;
     }
 
+    public DownloadRequest setStreamingDownload(boolean isStreamingEnabled) {
+        this.isStreamingEnabled = isStreamingEnabled;
+        return this;
+    }
+
+    public boolean isStreamingDownloadEnabled() {
```
Comment (smanikandan14) [inline ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadRequest.java]: Could we rename this api to `isResumable()` ? Also could add a method comment

File: ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadDispatcher.java [hunk 2]
```diff
@@ -30,40 +30,53 @@
...
+    private DownloadRequestQueue.CallBackDelivery mDelivery;
+    /**
+     * How many times redirects happened during a download request.
+     */
+    private int mRedirectionCount = 0;
     private long mContentLength;
-    private long mCurrentBytes;
     private boolean shouldAllowRedirects = true;
 
+    private long mDownloadedCacheSize = 0;
```
Comment (smanikandan14) [inline ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadDispatcher.java]: A comment for `mDownloadedCacheSize`

File: ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadRequest.java (hunk 1 above)
Comment (praveen2gemini) [reply ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadRequest.java]: Done: Changed to `isResumable()`

File: ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadDispatcher.java (hunk 2 above)
Comment (praveen2gemini) [reply ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadDispatcher.java]: A comment has been added to this variable, That will describe shortly about this variable that will tell the reason of variable why it appears here.

**Comment by smanikandan14** [comment]
Changes looks fine to me.

**Pull request merged**